bindings/python/test/environment/gravitational/test_earth.py

```diff
@@ -0,0 +1,93 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Physics
+# @file           bindings/python/test/objects/gravitational/test_earth.py
+# @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import pytest
+import numpy as np
+
+from ostk.core.filesystem import Path
```

Comment:
Same as the above, it seems `Path`, `URL`, `Earth` and `EarthGravitationalModelManager` are unused